src/query-builder/InsertQueryBuilder.ts

```diff
@@ -370,7 +370,12 @@ export class InsertQueryBuilder<Entity> extends QueryBuilder<Entity> {
 
                 if (Array.isArray(overwrite)) {
                     query += ` ${conflictTarget} DO UPDATE SET `;
-                    query += overwrite?.map(column => `${this.escape(column)} = EXCLUDED.${this.escape(column)}`).join(", ");
+                    query += overwrite?.map((val: string | {column: string, expression?: string}) => {
```

Comment:
wouldn't it apply `undefined` to the `query` if overwrite is undefined?

Reply:
okay, so `overwrite` seems to be defined here since its definitely an array. Then why do we need a question mark?

Reply:
I see it was before, but we need to correct this statement if its logically confuses.